### PR TITLE
Update Site title conditionally depending on deployment mode

### PR DESCRIPTION
**Pull request by bomanimc**
We are still using this repo to deploy two site: the main DUD portfolio and the new apps page. This PR updates things such that the Apps version of this repo deploys with the correct title.

### CODE REVIEW

File: gatsby-config.ts [hunk 1]
```diff
@@ -23,12 +25,13 @@ const config: GatsbyConfig = {
...
       options: {
-        name: "DIAL UP DIGITAL",
-        short_name: "DIAL UP DIGITAL",
+        name: isAppsSite ? "Nota" : "DIAL UP DIGITAL",
+        short_name: isAppsSite ? "Nota" : "DIAL UP DIGITAL",
         start_url: ".",
         background_color: "#ffffff",
         theme_color: "#000000",
         display: 'minimal-ui',
+        // TODO: Update so that deploys of the DUD portfolio site use the original DUD favicon.
```
Comment (bomanimc) [inline gatsby-config.ts:34]: We can probably fix this by making a small edit upon #64 to have to favicon files, `*_dud.*` and `*_nota.*`, that we select here conditionally depending on `isAppsSite`

File: gatsby-config.ts [hunk 2]
```diff
@@ -4,10 +4,12 @@ require("dotenv").config({
...
 });
 
+const isAppsSite = process.env.GATSBY_INDEX_PAGE_MODE == "apps";
+
 const config: GatsbyConfig = {
   siteMetadata: {
-    title: `DIAL UP DIGITAL`,
-    siteUrl: `https://dialup.digital`,
+    title: isAppsSite ? "Nota" : "Dial Up Digital",
+    siteUrl: isAppsSite ? "https://madebynota" : "https://dialup.digital",
```
Comment (jbirori) [inline gatsby-config.ts]: Should the first conditional result be `"https://madebynota.com"`

**Review by jbirori — COMMENTED** [review]
Looks pretty straightforward, just the one comment about the url

File: gatsby-config.ts (hunk 2 above)
Comment (bomanimc) [reply gatsby-config.ts]: Yep it should. Thanks!

Comment (bomanimc) [inline gatsby-config.ts]:
```suggestion
    siteUrl: isAppsSite ? "https://madebynota.com" : "https://dialup.digital",
```

**Comment by bomanimc** [comment]
@jbirori URL corrected

**Review by jbirori — APPROVED** [review]

**Pull request merged**